app/user/user-request-authorizer.js

```diff
@@ -35,7 +36,7 @@ const authorise = (request) => {
   }
 
   return userResolver
-    .getTokenDetails(bearerToken)
+    .getCachedUserDetails(bearerToken)
```

Comment:
The const `userResolver` will be either the cached or non-cached user-resolver. Currently the cached version has the `getCachedUserDetails()` method, but the non-cached has `getTokenDetails()` - so when the cache configuration is disabled it'll error every time as not able to find the function?